kcidb/templates/fluster_test.j2

```diff
@@ -40,11 +40,14 @@
             {% set statuses_counts = titles_statuses_counts.get(
                                         title, {"PASS": 0, "FAIL": 0}
                                      ) %}
+            {% set fluster_test_count_len = 5 %}
```

Comment:
You can calculate the counter length like this (not tested):
```jinja
{% set max_count = 0 %}
{% for _, titles_statuses_counts in boards_titles_statuses_counts.items() %}
{% for titles, statuses_counts in titles_statuses_counts.items() %}
{% set max_count = (statuses_counts.values() | list + [set_max_count]) | max %}
{% endfor %}
{% endfor %}
{% set field_width = (max_count | string | length + 1) | string %}
```

And then you can format each line like this:
```
(
    " ✅{0: >" + field_width + "}" +
    " ❌{1: >" + field_width + "}" +
    " Σ{2: >" + field_width + "}"
).format(
    statuses_counts.get("PASS", 0),
    statuses_counts.get("FAIL", 0),
    statuses_counts.values() | sum
)
```

Reply:
The length I set was based on the possible sub-tests count of fluster tests.
I'll make it dynamic. Thanks.

Reply:
Just using the dynamically-calculated field width wouldn't help. You also need alignment, which is done in the second part.